provisioners.proto

```diff
@@ -158,3 +159,24 @@ message SCEPProvisioner {
 message NebulaProvisioner {
 	repeated bytes roots = 1;
 }
+
+message Webhook {
+	string name = 1;
+	string id = 2;
+	string url = 3;
+	enum Kind {
+		NO_KIND = 0;
+		ENRICHING = 1;
+	}
+	Kind kind = 4;
+	string secret = 5;
+
+	message BasicAuth {
+		string username = 1;
+		string password = 2;
+	}
```

Comment:
Adding this into the Webhook makes the Go name `Webhook_BasicAuth`, the same happens for `Webhook_BearerToken`. This is ok, but I think we can move those to it's own type, so you have just `BasicAuth` and `BearerToken` in Go.

Reply:
8c2605f